### PR TITLE
KYLIN-4146 Add doc for KYLIN-4114

### PR DESCRIPTION
**Pull request by weibin0516**

### CODE REVIEW

**Comment by hit-lacus** [comment]
Thank you @weibin0516 , I will try to check this in Monday.

File: website/_docs/install/kylin_docker.cn.md [hunk 1]
```diff
@@ -3,8 +3,141 @@ layout: docs
 title:  "用 Docker 运行 Kylin"
 categories: install
 permalink: /cn/docs/install/kylin_docker.html
-version: v1.5.3
-since: v1.5.2
+since: v3.0.0-SNAPSHOT
```
Comment (hit-lacus) [inline website/_docs/install/kylin_docker.cn.md]: Nonexist version.

File: website/_docs/install/kylin_docker.cn.md [hunk 2]
```diff
@@ -3,8 +3,141 @@ layout: docs
...
+- Kylin
+
+并自动运行 `$KYLIN_HOME/bin/sample.sh ` 及在 Kafka 中创建 kylin_streaming_topic topic 并持续向该 topic 中发送数据。这是为了让用户启动容器后，就能体验以批和流的方式的方式构建 Cube 并进行查询。
+
+容器启动后，我们可以通过 docker exec 命令进入容器内。当然，由于我们已经将容器内指定端口映射到本机端口，我们可以直接在本机浏览器中打开各个服务的页面，如：
+
+- Kylin 页面：[http://127.0.0.1:7070/kylin/login](http://127.0.0.1:7070/kylin/login)
+- Hdfs NameNode 页面：[http://127.0.0.1:50070](http://127.0.0.1:50070/)
+- Yarn ResourceManager 页面：[http://127.0.0.1:8088](http://127.0.0.1:8088/)
+- HBase 页面：<http://127.0.0.1:60010>
```
Comment (hit-lacus) [inline website/_docs/install/kylin_docker.cn.md]: It is strange here, without correct mark for url.

File: website/_docs/install/kylin_docker.cn.md [hunk 3]
```diff
@@ -3,8 +3,141 @@ layout: docs
...
+- Kafka 1.1.1
+- Mysql
+- Maven 3.6.1
+
+## 快速试用 Kylin
+
+我们已将面向用户的 Kylin 镜像上传至 docker 仓库，用户无需在本地构建镜像，直接执行以下命令从 docker 仓库 pull 镜像：
+
+{% highlight Groff markup %}
+docker pull apachekylin/apache-kylin-standalone
```
Comment (hit-lacus) [inline website/_docs/install/kylin_docker.cn.md]: Could you please tell user that he can specific Kylin version by image's tag?

File: website/_docs/install/kylin_docker.md [hunk 4]
```diff
@@ -3,8 +3,141 @@ layout: docs
...
+{% endhighlight %}
+
+When the container starts, execute the docker exec command to enter the container. The source code is stored in the container dir `/home/admin/kylin_sourcecode`, execute the following command to package the source code: 
+
+{% highlight Groff markup %}
+cd /home/admin/kylin_sourcecod
+build/script/package.sh
+{% endhighlight %}
+
+After the package is complete, an installation package ending in `.tar.gz` will be generated in the `/home/admin/kylin_sourcecode/dist` directory, such as `apache-kylin-3.0.0-alpha2-bin-hbase1x.tar.gz`. We can use this installation package to deploy and launch Kylin services such as:
```
Comment (hit-lacus) [inline website/_docs/install/kylin_docker.md]: I am not sure, because I am not good at English. But I think *installation package* is strange in my mind, maybe binary package is better?

File: website/_docs30/install/kylin_docker.cn.md [hunk 5]
```diff
@@ -1,10 +1,143 @@
 ---
-layout: docs30
+layout: docs
```
Comment (hit-lacus) [inline website/_docs30/install/kylin_docker.cn.md]: Why did you change this?

File: website/_docs30/install/kylin_docker.md [hunk 6]
```diff
@@ -1,10 +1,143 @@
...
+In order to allow Kylin to build the cube smoothly, the memory resource we configured for Yarn NodeManager is 6G, plus the memory occupied by each service, please ensure that the memory of the container is not less than 8G, so as to avoid errors due to insufficient memory.
+
+For the resource setting method for the container, please refer to:
+
+- Mac user: <https://docs.docker.com/docker-for-mac/#advanced>
+- Linux user: <https://docs.docker.com/config/containers/resource_constraints/#memory>
+
+---
+
+Old docker image please check the github project [kylin-docker](https://github.com/Kyligence/kylin-docker/).
```
Comment (hit-lacus) [inline website/_docs30/install/kylin_docker.md]: As far as I see, I don't think this sentence is not fluent in terms of English grammar.

Maybe "For old docker image, please check the github page ...." will be better.

**Comment by weibin0516** [comment]
@hit-lacus , thanks for the review code, all comments resolved, please check again.
